packages/finicity-adapter/src/adapter.ts

```diff
@@ -115,8 +115,9 @@ export class FinicityAdapter implements WidgetAdapter {
     return connection;
   }
 
-  async DeleteConnection(id: string): Promise<ApiResponse> {
+  async DeleteConnection(id: string, userId: string): Promise<ApiResponse> {
     await this.cacheClient.set(id, null);
+    await this.DeleteUser(userId);
```

Comment:
I didn't see an option to delete an `institutionLoginId`, so it seems like you have to delete the customer entirely.